src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigation.java

```diff
@@ -18,7 +18,7 @@
 
     private static final String KEY = "INV";
     private static final String QUALIFIER = "MQ";
-    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    public static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
```

Comment:
Stylecheck doesn't complain here? In theory public should be before private

Reply:
No, because static fields that depend on other static fields must be ordered according to those dependencies.
![image](https://user-images.githubusercontent.com/38751994/108380226-8dd35000-71fe-11eb-94a7-db3a4d6d7739.png)